sky/cli.py

```diff
@@ -2109,6 +2112,11 @@ def storage_delete(names: Tuple[str], all: bool):  # pylint: disable=redefined-b
         click.echo('Deleting all storage objects.')
         storages = sky.storage_ls()
         names = [s['name'] for s in storages]
+    else:
+        names = [
+            name for name in _get_glob_clusters(names)
```

Comment:
`get_glob_cluster` gets all the clusters (not storages) that fit the criteria. More likely than not, need to create a new method `_get_glob_storages`

Reply:
Thanks for the catch!